### PR TITLE
Add Homebrew

### LINKED ISSUE #3366: Add homebrew icon
Hi! 

Got another shields.io related request (gotta love badges 😄)

**Name:** Homebrew
**Website:** http://brew.sh
**Alexa rank:** [23598](https://www.alexa.com/siteinfo/brew.sh) 
  

**Official resources for icon and color:**
  

- [brew.svg](https://github.com/Homebrew/brew.sh/tree/master/assets/img/brew.svg)
- [homebrew.svg](https://github.com/Homebrew/brew.sh/tree/master/assets/img/homebrew.svg)

The former isn't the one used on brew.sh but it might be a better one as [Brew](https://github.com/Homebrew/brew) now hosts both Homebrew and Linuxbrew. What would you say @MikeMcQuaid?

### PR DESCRIPTION
**Pull request by ddelange**
Closes #3366

image

**Issue:** #3366 
**Alexa rank:** [23598](https://www.alexa.com/siteinfo/brew.sh) 
  

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description

Hex taken directly from the original SVG

Special thanks to @PeterShaggyNoble for many insights and awesome support!

### CODE REVIEW

**Comment by PeterShaggyNoble** [comment]
Thanks for the PR, @ddelange :+1:

As this is based on a path I created, we'll need one of the other @simple-icons/maintainers to review the SVG before this can be merged. Reviewers: Note that this _does_ not line up perfectly with the source file due to issues Inkscape has with converting strokes to paths but an official representative of the project has given their OK on that in the original issue.

On the colour, I'd have gone with the `#2e2a24` used as the background colour on [their website](https://brew.sh/) but, not being overly familiar with the brand other than in name, I'll leave it to someone who does know it better to make that decision.

For GitHub source URLs, we use the latest commit of a file which, in this case, would be https://github.com/Homebrew/brew.sh/blob/4429abe3068444ac63eda2c128d2c9029145dbae/assets/img/homebrew.svg

**Comment by ddelange** [comment]
Personaly I would go with one of the yellows for the icon color, otherwise the resulting badge icon would be invisible without explicitly adding e.g. `&logoColor=white`. cc @MikeMcQuaid 

Example:
```
[![pre-commit](https://img.shields.io/badge/pre--commit-enabled-brightgreen?logo=pre-commit&logoColor=white)](https://github.com/pre-commit/pre-commit)
[![pre-commit](https://img.shields.io/badge/pre--commit-enabled-brightgreen?logo=pre-commit)](https://github.com/pre-commit/pre-commit)
```
[pre-commit](https://github.com/pre-commit/pre-commit)
[pre-commit](https://github.com/pre-commit/pre-commit)

**Comment by MikeMcQuaid** [comment]
Yeh, ideally the text and background colour would respect the website ones.

**Comment by ericcornelissen** [comment]
> Personally I would go with one of the yellows for the icon color, otherwise the resulting badge icon would be invisible without explicitly adding e.g. &logoColor=white

Unfortunately, we are a bit at odds with the shields.io here because we aim to provide a color that most represents the brand at hand, not a color that looks good on badges (and perhaps is related to the brand as well).


> Yeh, ideally the text and background colour would respect the website ones.

@MikeMcQuaid, can you elaborate? Which color from the website ([this website I guess?](https://brew.sh/)) would you recommend we use here?

**Comment by MikeMcQuaid** [comment]
> Which color from the website ([this website I guess?](https://brew.sh/)) would you recommend we use here?

Yes, that one. If it has to be two-tone: the font and background colours. If it doesn't: the same colour as the original icon.

File: _data/simple-icons.json [hunk 1]
```diff
@@ -2795,6 +2795,11 @@
             "hex": "41BDF5",
             "source": "https://github.com/home-assistant/home-assistant-assets"
         },
+        {
+            "title": "Homebrew",
+            "hex": "FBB040",
+            "source": "https://github.com/Homebrew/brew.sh/tree/master/assets/img/homebrew.svg"
```
Comment (runxel) [inline _data/simple-icons.json:2801]: See [Peters comment](https://github.com/simple-icons/simple-icons/pull/3395#issuecomment-668471766) on that.
```suggestion
            "source": "https://github.com/Homebrew/brew.sh/blob/2e576aaca83e62dda41a188597bb4bd20e75e385/assets/img/homebrew.svg"
```
(Protip: you can get that URL by typing "Y" while browsing any Github file)

File: _data/simple-icons.json [hunk 2]
```diff
@@ -2795,6 +2795,11 @@
             "hex": "41BDF5",
             "source": "https://github.com/home-assistant/home-assistant-assets"
         },
+        {
+            "title": "Homebrew",
+            "hex": "FBB040",
```
Comment (runxel) [inline _data/simple-icons.json:2800]: The color is a good choice! Leave it like that :)

**Comment by ddelange** [comment]
Right, should've taken some time to get a permalink (in other contexts I actually like the latest tag as pseudo permalink as it's shorter :smile:). If you have a protip for that too @runxel, hit me up! 

Thanks everyone! 💥

**Pull request merged**